src/utils/SiriusAdapter.cpp

```diff
@@ -381,7 +432,7 @@ class TOPPSiriusAdapter :
       csifile.store(out_csifingerid, csi_result);
     }
 
-    //clean tmp directory if debug level < 2
+    // clean tmp directory if debug level < 2
     if (debug_level_ >= 2)
```

Comment:
Consider using the functions in TOPPBase. E.g. makeAutoRemoveTempDirectory that will be removed if they go out of scope.

Reply:
Will add that next time.